### PR TITLE
chore: Setup `lint-staged`, bump `husky` and `prettier`.

### PR DESCRIPTION
**Pull request by avocadowastaken**
It's first of multiple PRs to integrate `prettier` formatting for all supported files.

See: https://github.com/prisma/docs/pull/1985#pullrequestreview-700364210

### CODE REVIEW

File: .husky/.gitignore [hunk 1]
```diff
@@ -1 +0,0 @@
-_
```
Comment (avocadowastaken) [inline .husky/.gitignore:1]: Husky 7 keeps `.gitignore` file inside of `_` dir.

File: .husky/_/husky.sh [hunk 2]
```diff
@@ -1,30 +0,0 @@
-#!/bin/sh
```
Comment (avocadowastaken) [inline .husky/_/husky.sh:1]: This file should be ignored.

File: .prettierignore [hunk 3]
```diff
@@ -2,3 +2,4 @@ node_modules/
 public/
 .cache/
 static/
+out/
```
Comment (avocadowastaken) [inline .prettierignore:5]: I moved if from `format` script for consistency.

**Comment by nilubava** [comment]
@umidbekk This looks good :)

**Pull request merged**